src/main/features/eligibility/model/eligibility.ts

```diff
@@ -120,7 +120,7 @@ export class Eligibility {
       this.claimantAddress === YesNoOption.YES &&
       this.defendantAddress === YesNoOption.YES &&
       this.eighteenOrOver === YesNoOption.YES &&
-      this.defendantAge === (DefendantAgeOption.YES || DefendantAgeOption.COMPANY_OR_ORGANISATION) &&
+      (this.defendantAge === (DefendantAgeOption.YES) || this.defendantAge === (DefendantAgeOption.COMPANY_OR_ORGANISATION)) &&
```

Comment:
Can we have some test for it?

We are fixing this issue for the second time. Sadly the first fix passed through the code review.

Reply:
The tests passed for this before and after this change. Im looking into it to try to get some better tests written